tests/wikipedia_transform/conftest.py

```diff
@@ -0,0 +1,120 @@
+import os
+from pathlib import Path
+
+import pytest
+
+from wikipedia_transform import WikipediaTransform
+from wikipedia_transform.embedding_pipelines import OpenAiEmbeddingPipeline
+from wikipedia_transform.generative_ai_pipelines import OpenAiPipeline
+from wikipedia_transform.models import wikipedia
+from wikipedia_transform.models.types import (
+    EnhancementType,
+    ModelResponse,
+    RecordKey,
+    RecordType,
+)
+from wikipedia_transform.readers import WikipediaReader
+
+
+@pytest.fixture(scope="session")
+def skip_if_ci() -> None:
```

Comment:
Nit: usually here I'd have an OpenaiSettings fixture rather than putting the OpenAI settings directly in your Settings.

Reply:
Suggest `OpenaiSettings` fixture like we discussed.